chain/cosmos/codec.go

```diff
@@ -25,6 +26,7 @@ func DefaultEncoding() simappparams.EncodingConfig {
 	banktypes.RegisterInterfaces(cfg.InterfaceRegistry)
 	ibctypes.RegisterInterfaces(cfg.InterfaceRegistry)
 	transfertypes.RegisterInterfaces(cfg.InterfaceRegistry)
+	wasmtypes.RegisterInterfaces(cfg.InterfaceRegistry)
```

Comment:
I'm not sure if we want these in the `DefaultEncoding` since not all cosmos chains are wasm. I could be convinced otherwise though.

Reply:
To keep in line with all the contract functions being in CosmosChain, I think they should be in there for now until we decide how to extend CosmosChain for wasm-supported chains.